bitcoin/docker-compose.yml

```diff
@@ -3,13 +3,17 @@ version: "3"
 services:
   bitcoind:
     container_name: bitcoind
+    image: ghcr.io/keep-network/local-setup/bitcoind
+    # build: ./bitcoind
```

Comment:
Can we get rid of the commented out `# build` lines?

Reply:
I couldn't decide what's better. I left the commented lines for convenience if we want to test changes in any module configuration.